www/artists/afyfe/gallery.html

```diff
@@ -24,24 +24,27 @@
             </div>
             <div id="gallery">
                 <p>
-                    <a href="mossyrocks.html"><img src="images/mossyrocks.jpg" alt="Mossy Rocks" class='b1'></a> <a href="natureresting.html"><img src="images/natureresting.jpg" alt="Nature Resting" class='b1'></a>
+                    <a href="mossyrocks.html"><img src="images/mossyrocks.jpg" alt="Mossy Rocks" class="b1"></a> <a href="natureresting.html"><img src="images/natureresting.jpg" alt="Nature Resting" class="b1"></a>
                     <br>
-                    Mossy Rocks | Nature Resting
+                    Mossy Rocks | Nature Resting (psychedelic ambiguity of nature and self.)
                 </p>
                 <p>
-                    <a href="atthebeach.html"><img src="images/atthebeach.jpg" alt="At the Beach" class='b1'></a> <a href="morning.html"><img src="images/morning.jpg" alt="Morning" class='b1'></a>
+                    <a href="atthebeach.html"><img src="images/atthebeach.jpg" alt="At the Beach" class="b1"></a> <a href="morning.html"><img src="images/morning.jpg" alt="Morning" class="b1"></a>
                     <br>
                     On the Beach | Morning
                 </p>
                 <p>
-                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class='b1'></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class='b1'></a>
+                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class="b1"></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class="b1"></a>
                     <br>
                     Prarie Lily | The Reflection Nebula
                 </p>
                 <p>
-                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.jpg" width="500" height="375" alt="no alt text"></a>
+                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.JPG" width="500" height="375" alt="no alt text"></a>
```

Comment:
Why did you change all of the .jpg to .JPG? We specifically changed them the other way because we renamed all of the photos because we don't want capitals in our filenames. Is your local branch malfunctioning?

Reply:
Possibly, i can fix that later. The text i added from the internet and the text before and after images are exactly what wayne told me to put. How would you like me to fix that?